Components/Button.ux

```diff
@@ -0,0 +1,71 @@
+<Panel ux:Class="Material.Button" Alignment="Center" Enabled="true" Raised="true" Height="48" MinWidth="88" Padding="8,0" HitTestMode="LocalBoundsAndChildren" BackgroundColor="White" InkColor="#ccc">
+
+	<string ux:Property="Text" />
+	<bool ux:Property="Enabled" />
+	<bool ux:Property="Raised" />
+	<float4 ux:Property="BackgroundColor"/>
+	<float4 ux:Property="InkColor"/>
+
+	<Rectangle ux:Name="RaisedButton_body" ZOffset="1" Height="36" Padding="8,0" CornerRadius="2">
```

Comment:
When assigning a `ux:Name` to a node, the value should start with a lowercase letter and be in camelCase. This here, for example, should be `ux:Name="raisedButtonBody"`.